core/src/main/java/com/yahoo/ycsb/generator/ConstantIntegerGenerator.java

```diff
@@ -36,7 +36,12 @@ public int nextInt() {
 		return i;
 	}
 
-	@Override
+    public long nextLong()
```

Comment:
IntegerGenerator doesn't have a nextLong method, so it's a bit odd for these other subclasses to include a nextLong.